roles/subman/tasks/main.yml

```diff
@@ -0,0 +1,140 @@
+---
+
+# subman_cpu_cpu fact
+- name: gather subman.cpu.cpu(s) fact
+  raw: subscription-manager facts --list | grep '^cpu.cpu(s).' | sed -n -e 's/^.*cpu.cpu(s).\s*//p'
+  register: internal_subman_cpu_cpu_cmd
+  ignore_errors: yes
+  become: yes
+  when: 'internal_have_subscription_manager'
+
+- name: extract result value for subman.cpu.cpu(s)
+  set_fact:
+    subman_cpu_cpu: "{{  internal_subman_cpu_cpu_cmd['stdout'] | trim | default('error') if internal_have_subscription_manager else 'N/A (subscription-manager not found)' }}"
```

Comment:
I'd suggest updating to the following to drop the 'error' strings:
`subman_cpu_cpu: "{{  internal_subman_cpu_cpu_cmd['stdout'] | trim | default(None) }}"`